### PR TITLE
fix: bump sdk version to 3.1.1

### PR DESCRIPTION
**Pull request by FelixTing**
fix: #497 

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-3.0.2
```
Comment (lenny-intel) [inline VERSION:1]: Why 3.1.1 and not 3.1.0?

Comment (lenny-intel) [reply VERSION:1]: Ok, I think I see that this is in prep for the future version.

Comment (cloudxxx8) [reply VERSION:1]: right, we missed the version number change step in v3.1.0, and we wrote the known bug on wiki:
https://wiki.edgexfoundry.org/pages/viewpage.action?pageId=88212172#Napa(NextRelease)-KnownBugs
it only implacts the logs, and it should be a blocker for release
it's the last one we need to manually change the version number.

Comment (lenny-intel) [reply VERSION:1]: > it only implacts the logs, and it should be a blocker for release

@cloudxxx8 , I assume you mean "it should **not** be a blocker for release", which I agree.

Comment (cloudxxx8) [reply VERSION:1]: right, should **not**

**Review by lindseysimple — APPROVED** [review]
LGTM

**Pull request merged**